src/main/resources/META-INF/resources/primefaces/keyboard/keyboard.js

```diff
@@ -961,6 +961,8 @@ PrimeFaces.widget.KeyboardUtils = {
 	},
 
 	createLayoutFromTemplate : function(template) {
+	    // GitHub #3487: Unicode conversion
```

Comment:
PrimeFaces uses tabs (tab character) for indenting code, please convert.

Reply:
Also maybe the comment about github ticket is maybe unwanted.

Reply:
Comments get stripped when the library is built so they are harmless to be in there.  I actually find them useful when a piece of code looks strange to go back and look at the ticket.

Reply:
Also I fixed the spaces to tabs

Reply:
Nice, thank you. :-) No mixing of indenting styles ...